EIPS/eip-4804.md

```diff
@@ -53,7 +53,7 @@ where
 
 ### Resolve Mode
 
-Once the "To" address and chainid are determined, the protocol will check the resolver mode of contract by calling "resolveMode" method. The protocol currently supports two resolve modes:
+Once the "To" address and chainid are determined, the protocol will check the resolver mode of contract by calling "resolveMode() returns (bytes32)" method. The protocol currently supports two resolve modes:
 
 #### Manual Mode
 
```

Comment:
I'm kinda just assuming what the return value is here:

```suggestion
The `resolveMode` return value indicating manual mode MUST be `0x00000000000000000000000000000000000000000000000000006d616e75616c`.

```

Should do something similar for auto mode, maybe with `0x000000000000000000000000000000000000000000000000000000006175746f`.

Reply:
Thanks for the comment.  I will update this accordingly.